pkg/bot/slack.go

```diff
@@ -139,24 +138,25 @@ func (sm *slackMessage) HandleMessage(b *SlackBot) {
 				return
 			}
 			// Serve only if current channel is in config
-			if b.ChannelName == info.Name {
-				sm.IsAuthChannel = true
-			}
 		}
 	}
 	// Serve only if current channel is in config
-	if b.ChannelName == sm.Event.Channel {
-		sm.IsAuthChannel = true
+
+	var Accessbinding config.Accessbinding
```

Comment:
```suggestion
	accessBinding := config.AccessBinding{}
```
same for all other places